support-frontend/assets/pages/digital-subscription-landing/digitalSubscriptionLandingReducer.js

```diff
@@ -14,8 +14,11 @@ export type State = {
   }
 };
 
+const { orderIsAGift } = window.guardian;
```

Comment:
Should we add a default for `orderIsAGift` incase `window.guardian.orderIsAGift` is `undefined`?

`const { orderIsAGift = false } = window.guardian;`

Reply:
My understanding is that this value always gets set. But I can add a default if you think that would be more secure?

Reply:
yes It's probably safer to have the default, just incase for some reason `window.guardian.orderIsAGift` is `undefined` (someone changes the server-side template for example). The code will probably work in that scenario as `undefined` is still falsey, but if we're actually checking for a `boolean` value at some point it might fall over.